frontend/assets/l10n/app_de.json

```
@@ -96,6 +96,7 @@
     "cameraAccessRequiredSettings": "Um einen QR-Code einzuscannen, benötigt die App Zugriff auf die Kamera. In den Einstellungen können Sie der App den Zugriff auf die Kamera erlauben.",
     "cardAlreadyActivated": "Der eingescannte QRCode wurde bereits aktiviert.",
     "cardExpired": "Ihre Karte ist abgelaufen. Unter \"Weitere Aktionen\" können Sie einen Antrag auf Verlängerung stellen.",
+    "cardExpiredBeforeActivation": "Ihre Karte ist bereits abgelaufen und kann nicht mehr aktiviert werden. Bitte beantragen Sie sie erneut.",
     "cardInvalid": "Ihre Karte ist ungültig. Sie wurde entweder widerrufen oder auf einem anderen Gerät aktiviert.",
```

🔧 Since we are now explicitly check for `revoked` activation state, the message should be adjusted
```suggestion
    "cardInvalid": "Ihre Karte wurde nicht gefunden.",
```
@connium this now only happens if the card can not be found in our database, which will likely not happen except someone adjusted the qr code and sneaked in different data